app/views/dashboard/epic_queues/_epic_queue_list.html.haml

```diff
@@ -20,18 +20,24 @@
 .panel.panel-default#epic-queue-panel
```

Comment:
Can you remove the panel and the heading/body since they no longer exist with Bootstrap 4 and replace it with a column div with `mx-auto` to center it? A div with `col-12 col-sm-10 mx-auto` is what the notifications page is using.